src/transaction/boot_sr.h

```diff
@@ -85,6 +85,17 @@ struct check_args
         (boot_Server_status == BOOT_SERVER_UP \
          || boot_Server_status == BOOT_SERVER_MAINTENANCE)
 
+extern bool boot_Enabled_flush_daemons;
+#define BO_IS_FLUSH_DAEMON_AVAILABLE() \
+        (boot_Enabled_flush_daemons == true)
+#define BO_ENABLE_FLUSH_DAEMONS() \
+        do \
+          { \
+            assert (!BO_IS_FLUSH_DAEMON_AVAILABLE ());\
+            boot_Enabled_flush_daemons = true;\
+          } \
+        while (0);
+
```

Comment:
#if defined(SERVER_MODE) ~ #endif 추가 및 do ~ while (0)에서 ; 제거
```suggestion
#if defined(SERVER_MODE)
extern bool boot_Enabled_flush_daemons;
#define BO_IS_FLUSH_DAEMON_AVAILABLE() \
        (boot_Enabled_flush_daemons == true)
#define BO_ENABLE_FLUSH_DAEMONS() \
        do \
          { \
            assert (!BO_IS_FLUSH_DAEMON_AVAILABLE ());\
            boot_Enabled_flush_daemons = true;\
          } \
        while (0)
#endif
```

Reply:
수정했습니다.